imports/ui_3/views/components/vote-cast-panel.js

```diff
@@ -44,9 +45,11 @@ function castVoteBasedOnPermission(topicId, castedVote, callback) {
 Template.Vote_cast_panel.viewmodel({
   topic: undefined,
   temporaryVote: undefined,
+  partnerId: undefined,
   onCreated(instance) {
     instance.firstRun = true;
     this.topic(instance.data);
+    this.partnerId(Meteor.user().partnerId(Session.get('activeCommunityId')));
```

Comment:
Ezt a sort, ha mar reaktiv valtozo,  szerintem erdemes lenne autorun-ba tenni. Mert van az elejen egy pillanat amikor az activeCommunity meg epp toltodik be, ha akkor hivodik meg, akkor undefined lesz es tobbet nem fut le hogy felvegye a helyes erteket.